packages/uikit-react-native/src/hooks/useMentionSuggestion.ts

```diff
@@ -89,7 +89,7 @@ const useMentionSuggestion = (params: {
         .then((members) => members.slice(0, mentionManager.config.suggestionLimit));
     } else {
       return freshChannel.members
-        .sort((a, b) => a.nickname?.localeCompare(b.nickname))
+        .sort((a, b) => a.nickname?.toLowerCase().localeCompare(b.nickname.toLowerCase()))
```

Comment:
Can you add the relevant details we discussed here as a Note comment?
```
// NOTE: blah-blah
```

Reply:
I have added a comment.